requirements/frozen.txt

```diff
@@ -45,6 +45,7 @@ google-api-python-client==1.6.2
 gunicorn==19.7.1
 html5lib==0.999999999
 httplib2==0.10.3
+humanfriendly
```

Comment:
This needs to be locked to a version, we switched from requirements to frozen requirements.